siaskynet/convert.py

```diff
@@ -0,0 +1,19 @@
+"""Skynet convert API.
+"""
+
+
+from . import utils
+
+
+def __default_convert_options():
+    """Returns the default convert options."""
+
+    obj = utils.__default_options("/skynet/skyfile")
```

Comment:
is this correct endpoint?

Reply:
Yeah, this is a double-purpose endpoint both for uploading and converting.